homeassistant/components/daikin/sensor.py

```diff
@@ -179,11 +214,18 @@ def __init__(
         """Initialize the sensor."""
         self.entity_description = description
         self._api = api
-        self._attr_name = f"{api.name} {description.name}"
+        self._attr_entity_registry_enabled_default = description.enabled_default
```

Comment:
We don't need to set `_attr_entity_registry_enabled_default` if we set `entity_description` with `entity_registry_enabled_default` set.